speechbrain/lobes/models/huggingface_transformers/discrete_ssl.py

```diff
@@ -284,3 +330,49 @@ def forward(
             org_tokens, SSL_layers, deduplicates, bpe_tokenizers
         )
         return org_tokens, org_embedding, processed_tokens
+
+    def decode(self, tokens, SSL_layers=None):
```

Comment:
Please add link from which this code is taken from (I guess https://github.com/speechbrain/benchmarks/blob/c87beb61d4747909a133d3e1b3a3df7c8eda1f08/benchmarks/DASB/Libri2Mix/separation/conformer/train_discrete_ssl.py#L44)
This is useful for back tracing in case of bugs.